### PR TITLE
Improving File Storage page

### PR DESCRIPTION
**Pull request by rockstardev**
Resolves: #2272

Improving navigation between files and storage services and rewording info text. Go at it @Zaxounette if you want to try and change my l33t text ;)

### CODE REVIEW

File: BTCPayServer/Views/Server/Files.cshtml [hunk 1]
```diff
@@ -5,48 +5,64 @@
...
-                        - <a asp-action="DeleteFile" asp-route-fileId="@file.Id">Remove</a>
-                    </td>
-                </tr>
-            }
-        </tbody>
-    </table>
+    <p>
+        Before being able to upload you first need to 
+        <a asp-action="storage" asp-route-forceChoice="true" asp-route-returnurl="@ViewData["ReturnUrl"]">
+            choose your external storage service provider
```
Comment (Zaxounette) [inline BTCPayServer/Views/Server/Files.cshtml:13]:
```suggestion
            choose your file storage service provider
```

**Review by Zaxounette — CHANGES_REQUESTED** [review]
Remove `external` from description since one of the options is an internal storage provider.
Replaces with `file` to better indicate that it's files that are stored.

**Review by pavlenex — APPROVED** [review]
tACK.

Agreed with @Zaxounette's suggestion above.

**Review by Kukks — APPROVED** [review]
LGTM, any reason why PR is a draft @rockstardev?

**Comment by pavlenex** [comment]
@Kukks I asked @rockstardev about it and it seems he wants to do a few more improvements that should be done early next week.

**Review by bolatovumar — APPROVED** [review]

**Comment by rockstardev** [comment]
OK, I'm done with my changes, needed to ensure that you can only select valid providers in dropdown and clean up those error sandwiches. See https://youtu.be/ortCV8U4kfQ from ~40:00 in if you want to understand changes more deeply... and see YouTube trollbox ofc.

File: BTCPayServer/Views/Server/Files.cshtml [hunk 2]
```diff
@@ -5,48 +5,64 @@
...
+            <span class="fa fa-question-circle-o text-secondary" title="More information..."></span>
+        </a>
+    </p>
 }
 else
 {
-    <p class="text-secondary mt-3">
-        There are no files yet.
+    <p>
+        Change your <a asp-action="storage" asp-route-forceChoice="true" asp-route-returnurl="@ViewData["ReturnUrl"]">external storage service</a> provider.
```
Comment (Zaxounette) [inline BTCPayServer/Views/Server/Files.cshtml:23]:
```suggestion
        Change your <a asp-action="storage" asp-route-forceChoice="true" asp-route-returnurl="@ViewData["ReturnUrl"]">file storage service</a> provider.
```

**Review by Zaxounette — CHANGES_REQUESTED** [review]
A second instance of `external` appeared.
Changes to `file`, same as previous review.

**Pull request merged**